compiler/src/tests/cxxfrontend.cc

```diff
@@ -1178,12 +1178,6 @@ class MiniGlueVisitor : public Visitor
                 Dsymbol *s = (*d->members)[i];
                 s->accept(this);
             }
-            ClassDeclarations aclasses;
-            for (size_t i = 0; i < d->members->length; i++)
-            {
-                Dsymbol *member = (*d->members)[i];
-                member->addLocalClass(&aclasses);
-            }
```

Comment:
Just so that it stays representative to what gdc/ldc would be doing to get the same information:
```d
            ClassDeclarations aclasses;
            getLocalClasses(d, &aclasses);
```

Reply:
ol